packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -17,10 +17,17 @@ function setProfile(awsConfig, profileName) {
   let credentials = {};
   let config = {};
   if (fs.existsSync(credentialsFilePath)) {
+    makeFileOwnerReadWrite(credentialsFilePath);
     credentials = ini.parse(fs.readFileSync(credentialsFilePath, 'utf-8'));
+  } else {
+    fs.openSync(credentialsFilePath, 'w', 0o600);
```

Comment:
Shouldn't you close the file?

Reply:
no, because we need to write to the file in the method `setProfile`

Reply:
yes, nice catch

Reply:
I don't see the opened file descriptor used in `writeSync`, so this file descriptor needs to be closed

Reply:
nice catch. 
fixed